wsi/physicaldisplay.cpp

```diff
@@ -586,6 +586,18 @@ bool PhysicalDisplay::GetDisplayConfigs(uint32_t *num_configs,
   return true;
 }
 
+void PhysicalDisplay::SetDisplayRenderingState(bool rendering)
```

Comment:
Though DisplayManager has an interface IgnoreUpdates to ignore Rendering content into DRM but status set by this interface will be affected by display hotplug events and behavior of this existing interface is not reliable. So we added a new interface to ensure the HWC content rendering skipping works as expected.

Reply:
hmm that interface was added only for this purpose, if hot plug events are impacting that we should fix that rather than adding a new interface.